flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/stream/compact/CompactOperator.java

```diff
@@ -68,7 +64,8 @@
  * the file system.
  */
 public class CompactOperator<T> extends AbstractStreamOperator<PartitionCommitInfo>
```

Comment:
Is this class supposed to be used by users? If not we should mark it as `@Internal`

Reply:
Do we need to mark `@Internal` all these classes as well? I thought that matters only for classes in `api` packages :smile:

Reply:
It'd be nice if the world was like that, but it isn't. I think any public classes should be annotated.